README.md

```diff
@@ -52,9 +52,11 @@ Put this chunk of code somewhere in the source code of your site.
 
 This may change in the future, so stay tuned ...
 
+Note: This code currently only works with the Flow Manager from [GPII's review4 branch](https://github.com/GPII/universal/tree/review4).
```

Comment:
Actually, it works with master too, the thing is that the solution entry must be included both in win32 and linux solutions registry entries, and such duplication is a side-effect of how web solutions are treated. The review4 branch includes support for passing web-based solutions to a local flowManager, and that's the reason it works for you with the review4 branch of universal and by having your solution entry inside the web solutions registry.
In any case, I don't see why this should be included as a note here, probably, this "implementation issue" should be put elsewhere in some other documentation/tutorial/wiki/whatever.